Lib/fontTools/feaLib/lexer.py

```diff
@@ -195,14 +192,14 @@ def __init__(self, featurefile):
     def __iter__(self):
         return self
 
-    def next(self, comments=False):  # Python 2
-        return self.__next__(comments)
+    def next(self):  # Python 2
+        return self.__next__()
 
-    def __next__(self, comments=False):  # Python 3
+    def __next__(self):  # Python 3
         while self.lexers_:
             lexer = self.lexers_[-1]
             try:
-                token_type, token, location = lexer.next(comments)
+                token_type, token, location = lexer.next()
```

Comment:
Could we change all occurrence of `lexer.next()` with `next(lexer)`?
I know, it works in both cases,  but using the latter idiom would make it appear less legacy python 2.

/cc @brawer 

```
$ git grep "\.next()"
Lib/fontTools/feaLib/lexer.py:202:                token_type, token, location = lexer.next()
Lib/fontTools/feaLib/lexer.py:207:                fname_type, fname_token, fname_location = lexer.next()
Lib/fontTools/feaLib/lexer.py:210:                #semi_type, semi_token, semi_location = lexer.next()
Lib/fontTools/feaLib/parser.py:1379:                 self.next_token_location_) = self.lexer_.next()
Lib/fontTools/voltLib/parser.py:598:             self.next_token_location_) = self.lexer_.next()
```